### PR TITLE
Adding Doom II and Heretic

### PR DESCRIPTION
**Pull request by ScootyPuffJr1**

### CODE REVIEW

File: games/DOOM II.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿DOOM II:
+  difficulty:
+    baby: 2
+    easy: 5
+    medium: 3
+  random_monsters: shuffle
+  random_pickups: shuffle
+  allow_death_logic: false
+  start_with_computer_area_maps: random
```
Comment (Daivuk) [inline games/DOOM II.yaml:9]: if for first timers to AP, but know doom, I would put this `false`
if for frist doom timers, put `true`.
not random.

File: games/Heretic.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿Heretic:
+  goal: complete_all_levels
```
Comment (Daivuk) [inline games/Heretic.yaml:2]: i would set that to boss levels only

File: games/DOOM II.yaml (hunk 1 above)
Comment (ScootyPuffJr1) [reply games/DOOM II.yaml:9]: I matched what we have in Doom 1993 for maps, but I don't think it matters too much.

File: games/Heretic.yaml (hunk 2 above)
Comment (Daivuk) [reply games/Heretic.yaml]: ya it was how it was before, but the all boss levels are a kinda newish feature.
in the end, probably doest matter. Ship it.

**Pull request merged**